### PR TITLE
PEP-354

### PR DESCRIPTION
**Pull request by adgomam**
[PEP-354](https://adgorithmics.atlassian.net/browse/PEP-354)

### CODE REVIEW

File: lib/peppermint_sdk.dart [hunk 1]
```diff
@@ -7,7 +7,10 @@ export 'src/models/notification/notif.dart';
 export 'src/models/profile/profile.dart';
 export 'src/peppermint_web3.dart';
 export 'src/peppermint_utility.dart';
+export 'src/peppermint_constants.dart';
```
Comment (adgojoe) [inline lib/peppermint_sdk.dart]: No need to export

File: lib/peppermint_sdk.dart [hunk 2]
```diff
@@ -7,7 +7,10 @@ export 'src/models/notification/notif.dart';
...
 export 'src/widgets/image_loader.dart';
+export 'src/widgets/button.dart';
+export 'src/widgets/popup.dart';
```
Comment (adgojoe) [inline lib/peppermint_sdk.dart]: No need to export.
Let them be used internally.

**Review by adgojoe — CHANGES_REQUESTED** [review]
remove flutter plugins
remove flutter-plugins-dependencies

File: lib/peppermint_sdk.dart (hunk 1 above)
Comment (adgomam) [reply lib/peppermint_sdk.dart]: but I used it in main

Comment (adgojoe) [reply lib/peppermint_sdk.dart]: then import in main, not export in here.

**Pull request merged**